src/App.js

```diff
@@ -1,24 +1,20 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import Header from "./components/Header";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import Home from "./components/Home";
```

Comment:
This line along with line 14 caused me issues - when I loaded your site I immediately hit an error, it looks like you have the root route (/) going to a component called 'Home' but there is no Home.jsx component in your components folder. If I manually typed in /articles in the address bar then I got the articles all loading.

Please could you fix this, you'll need decide either: What do you want your 'Home' component to display to the user, or do you want the root route (/) to just display all articles? Have a look